sidebars.js

```diff
@@ -115,6 +115,37 @@ const sidebars =
           ],
         },
       ],
+    },
+    {
+      type: 'category',
+      label: 'Resources',
+      link: 
+      {
+        type: 'generated-index',
+        title: 'Resources',
+        description: 'Resources created by the Organization to understand our workflow.',
+        keywords: ['organization, resources'],
+      },
+      collapsed: false,
+      items: 
+      [
+        {
+          type: 'category',
+          label: 'Workflows',
+          link: 
+          {
+            type: 'generated-index',
+            title: 'Workflows',
+            description: 'A workflow is a series of steps that are followed to design, develop, test, and release software. The software development workflow typically can involve multiple stages, each with its own set of tasks and deliverables.',
```

Comment:
mention that this is our organization workflow

Reply:
Made a new commit to reflect the changes requested.